hydra-node/src/Hydra/Chain/CardanoClient.hs

```diff
@@ -223,6 +244,13 @@ queryEraHistory :: NetworkId -> SocketPath -> QueryPoint -> IO (EraHistory Carda
 queryEraHistory networkId socket queryPoint =
   runQuery networkId socket queryPoint $ QueryEraHistory CardanoModeIsMultiEra
 
+-- | Query current era at given point.
+--
+-- Throws at least 'QueryException' if query fails.
+queryCurrentEra :: NetworkId -> SocketPath -> QueryPoint -> IO AnyCardanoEra
+queryCurrentEra networkId socket queryPoint =
+  runQuery networkId socket queryPoint $ QueryCurrentEra CardanoModeIsMultiEra
```

Comment:
Could: remove this after we made all era-specific queries use the `queryCurrentEraExpr` internally.